scene/3d/visibility_notifier.cpp

```diff
@@ -153,6 +155,20 @@ void VisibilityEnabler::_find_nodes(Node *p_node) {
 		}
 	}
 
+	{
```

Comment:
Nitpick, but I guess you could do the three animation related checks in the same scope so we save some vertical space. They're still semantically related so it makes sense IMO.

Reply:
Or even:
```
if (Object::cast_to<AnimationPlayer>(p_node) || Object::cast_to<AnimationTree>(p_node) || Object::cast_to<AnimationTreePlayer>(p_node)) {
	add = true;
}
```
But that's maybe not so readable.

Reply:
> Nitpick, but I guess you could do the three animation related checks in the same scope so we save some vertical space. They're still semantically related so it makes sense IMO.

Though that's consistent with the style used in `VisibilityNotifier2D` for `AnimationPlayer` and `AnimatedSprite`, so I guess that's fine :)

Reply:
Yeah I wasn't altogether convinced scoping was necessary here, but I just followed the convention in the earlier code. I don't really think it matters that much one way of the other.